core/src/avm1/property_decl.rs

```diff
@@ -128,6 +128,7 @@ impl Declaration {
 ///     "hidden" => string("shh!"; DONT_ENUM | DONT_DELETE | READ_ONLY);
 /// };
 /// ```
+#[allow(unused_macro_rules)]
```

Comment:
```suggestion
#[allow(unknown_lints, unused_macro_rules)]
```

You'll probably want this so that it doesn't warn on stable and beta.

Reply:
This is already covered by:
https://github.com/ruffle-rs/ruffle/blob/edad1569fbb235527ed3241b51e28fa323eca637/.cargo/config.toml#L11